app/Config/DocTypes.php

```diff
@@ -30,4 +30,15 @@ class DocTypes
         'xhtml-rdfa-1'      => '<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML+RDFa 1.0//EN" "http://www.w3.org/MarkUp/DTD/xhtml-rdfa-1.dtd">',
         'xhtml-rdfa-2'      => '<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML+RDFa 1.1//EN" "http://www.w3.org/MarkUp/DTD/xhtml-rdfa-2.dtd">',
     ];
+
+    /**
+     * Endline HTML Tag
+     *
+     * This setting is for html and form helpers.
+     *
+     * For :
+     * - `<input>`
+     * - `<img>`
+     */
+    public bool $xhtml = false;
```

Comment:
Following my comment above and the proposed code, I think this should be better renamed to `$html5` or `$html5compatible`.

Reply:
> to retain compatibility with older document types

But `/` is needed only in XHTML specifications.